lib/drivers/medtronic/processData.js

```diff
@@ -1083,6 +1089,11 @@ function buildTimeChangeRecords(records, mostRecent, type) {
 
   var timeChangeRecords = null;
 
+  function getCGMTimeChangeType(obj) {
+    var type = obj.date[1] >> 5 & 7;
```

Comment:
This data is used for other history types as well, and is called `subCode`. Not sure if you want to make it more generic now. Not a big deal, since you're only using it for the TimeChangeType at the moment.

Reply:
Good point. Will change it.